components/mediation-initializer/org.wso2.carbon.mediation.transport.handlers/src/main/java/org/wso2/carbon/mediation/transport/handlers/requestprocessors/swagger/format/SwaggerGenerator.java

```diff
@@ -142,47 +142,13 @@ protected String retrieveFromRegistry(API api, CarbonHttpRequest request) throws
                 resourcePathBuilder.append(":v").append(api.getVersion());
             }
             resourcePathBuilder.append("/swagger.json");
-            resourcePath = resourcePathBuilder.toString();
-
-        } else {
-            if (resourcePath.startsWith(CONFIG_REG_PREFIX)) {
-                resourcePath = resourcePath.substring(5);
-            } else {
-                resourcePath = resourcePath.substring(4);
-                isSourceConfigReg = false;
-            }
-        }
+            resourcePath = CONFIG_REG_PREFIX + resourcePathBuilder.toString();
```

Comment:
You can refactor removing this "CONFIG_REG_PREFIX" constant and use one defined in SwaggerProcessorConstants.java. Also refactor usage of CONFIG_REG_PREFIX in SwaggerGenerator.java

Reply:
+1 fixed